File: JavaScript/Faccat/Exercicio84.js
let A = [];
let B = [];
let S = [];

for (let indice = 0; indice < 5; indice++) {
    A[indice] = parseFloat(prompt("Digite um número para alocar no vetor A:"));
    B[indice] = parseFloat(prompt("Digite um número para alocar no vetor B:"));
    S[indice] = A[indice] + B[indice];
}

console.log("Vetor da soma dos vetores A e B:");
for (let indice = 0; indice < 5; indice++) {
    console.log(S[indice]);
}

File: Java/src/Faccat/Exercicio84.java
package Faccat;

import java.util.Scanner;

public class Exercicio84 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double[] A = new double[5];
        double[] B = new double[5];
        double[] S = new double[5];

        for (int indice = 0; indice < 5; indice++) {
            System.out.println("Digite um número para alocar no vetor A:");
            A[indice] = scanner.nextDouble();
            System.out.println("Digite um número para alocar no vetor B:");
            B[indice] = scanner.nextDouble();
            S[indice] = A[indice] + B[indice];
        }

        System.out.println("Vetor da soma dos vetores A e B:");
        for (int indice = 0; indice < 5; indice++) {
            System.out.println(S[indice]);
        }

        scanner.close();
    }
}
